sklearn/kernel_ridge/__init__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+# Authors: Mathieu Blondel <mathieu@mblondel.org>
+#          Jan Hendrik Metzen <jhm@informatik.uni-bremen.de>
+#          Carlos Perales <sir.perales@gmail.com>
+# License: BSD 3 clause
+
+from .krc import KernelRidgeClassification
+from .krr import KernelRidgeRegression
+
+KernelRidge = KernelRidgeRegression
+KRR = KernelRidgeRegression
```

Comment:
remove this please

Reply:
We don't generally have aliases like this

Reply:
Done